src/main/java/net/fabricmc/mappingio/format/intellij/MigrationMapFileReader.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2023 FabricMC
```

Comment:
Year need an update after reviving this PR after many months.

Reply:
Thanks, fixed!